app/models/card.rb

```diff
@@ -1,23 +1,41 @@
 class Card < ActiveRecord::Base
   before_validation :set_review_date, on: :create
+  before_save :set_downcase_text
   validate :original_translated_text_equal
   validates :original_text, :translated_text, :review_date,
             presence: { message: 'Необходимо заполнить поле.' }
 
-  protected
-    def set_review_date
-      self.review_date = Time.now + 3.days
+  def self.pending(id)
+    if id
+      where('review_date <= ? AND id = ?', Time.now, id).order('RANDOM()')
+    else
+      where('review_date <= ?', Time.now).order('RANDOM()')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.